Storage/ThirdParty/SwiftData.swift

```diff
@@ -219,6 +219,10 @@ private class SQLiteDBStatement {
         return nil
     }
 
+    func close() {
+        sqlite3_finalize(pointer)
+    }
+
     deinit {
         sqlite3_finalize(pointer)
```

Comment:
I'm getting a EXC_BAD_ACCESS here when running this in my iPhone 6 8.3 simulator. Could we be double closing the pointer from the added close call above?